### PR TITLE
A referenced id should be able to follow any reasonable convention, n…

### PR DESCRIPTION
**Pull request by dczulada**
…ot just a BSON objectId

### CODE REVIEW

**Comment by dczulada** [comment]
This will also need a go-cda-tools bundle update

File: lib/cypress/go_import.rb [hunk 1]
```diff
@@ -1,35 +1,11 @@
 module Cypress
   module GoImport
-    def self.update_conditions(record)
```
Comment (okeefm) [inline lib/cypress/go_import.rb]: Do we not need to update conditions anymore? Or are we doing this on the `cdatools` side?

Comment (dczulada) [inline lib/cypress/go_import.rb]: https://github.com/projectcypress/go-cda-tools/pull/28

Comment (okeefm) [inline lib/cypress/go_import.rb]: 👍

**Pull request merged**